lib/opencpu/client.rb

```diff
@@ -8,12 +8,29 @@ def initialize
     end
 
     def execute(package, function, options = {})
-      user   = options.fetch :user, :system
-      data   = options.fetch :data, {}
-      format = options.fetch :format, :json
-      github_remote = options.fetch :github_remote, false
+      user              = options.fetch :user, :system
+      data              = options.fetch :data, {}
+      format            = options.fetch :format, :json
+      github_remote     = options.fetch :github_remote, false
+      convert_na_to_nil = options.fetch :convert_na_to_nil, false
+
       process_query package_url(package, function, user, github_remote, :json), data, format do |response|
-        JSON.parse(response.body)
+        data =  JSON.parse(response.body)
+        data = convert_na_to_nil(data) if convert_na_to_nil
```

Comment:
I really don't like having a variable as the same name as the function. 
Also option-flags with a default of falsy, don't have to be be fetched into a variable first, though at that point a method description would be extra handy. 
options: 
 - convert_na_to_nil: if true all string values in the response that equal "NA" will be converted to nil.